server/runtime/usr/local/bin/start-server.sh

```diff
@@ -5,4 +5,12 @@ set -e
 # in /usr/local/bin/qgis_mapsev_wrapper)
 env | sed -e 's/^/export /' > /tmp/init_env.sh
 
+if [[ "${UID}" != 0 ]]
+then
+    echo "Switching listen port to 8080"
+    cd /tmp
+    sed -i -e 's/<VirtualHost \*:80>/<VirtualHost *:8080>/' /etc/apache2/sites-available/000-default.conf
+    sed -i -e 's/Listen 80/Listen 8080/' /etc/apache2/ports.conf
+fi
```

Comment:
Shouldn't we add some doc in the readme for that?

Reply:
OK, adding that.